package.json

```diff
@@ -4,6 +4,8 @@
   "description": "Wrapper for ActiveDirectory that gets the group for the username passed in through the x-iisnode-auth_user header",
   "main": "index.js",
   "scripts": {
+    "lint": "standard --verbose | snazzy",
```

Comment:
It's already not global. npm is kind enough to use local if available in npm scripts so no need to put in long ./node_modules/.bin type paths.